Use a different set of flags when invoking df on FreeBSD.
https://sourceforge.net/p/xdiskusage/git/ci/275754ad0a24716d86da1aaaf452b0906211e859/
--- xdiskusage.C.orig	2015-02-09 17:29:18.000000000 -0600
+++ xdiskusage.C	2020-06-18 18:11:19.000000000 -0500
@@ -28,10 +28,13 @@
 #elif defined(__bsdi__)
 # define DF_COMMAND "df"
 # define DU_COMMAND "du -x"
+#elif defined(__FreeBSD__)
+# define DF_COMMAND "df -k -t noprocfs,devfs,fdescfs"
+# define DU_COMMAND "du -kx"
 #elif defined(SVR4) || defined(__sun)
 # define DF_COMMAND "/bin/df -k"
 # define DU_COMMAND "/bin/du -kd"
-#else // linux and irix
+#else // linux
 # define DF_COMMAND "df -k -x usbfs -x tmpfs"
 # define DU_COMMAND "du -kx"
 #endif
